lib/players/human.rb

```diff
@@ -0,0 +1,7 @@
+class Human < Player
+
+	 def move(board)
+	 	input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  gets.chomp
end
```